src/pudl/transform/ferc1.py

```diff
@@ -1054,32 +1105,6 @@ def _calculation_components_subtotal_calculations(
     return calc_comps_w_totals
 
 
-def _check_subtotal_calculations(
```

Comment:
now both the "standard" calcs and the subtotal calcs are being checked via `reconcile_one_type_of_table_calculations`